### ISSUE TITLE
Multi-Channel output LV2 version?

### ISSUE THREAD

**Issue by unfa** [zynaddsubfx/zyn-fusion-issues#220]
I've noticed it's possibleto run Zyn-Fusion in multi-output mode.

I wonder if there could be a plug-in version that uses that?

It could be very useful for drumkits. I stopped using drumkits in ZynAddSubFX, because without the option to process each instrument in a kit separately - I can't get teh sound I want, but with multi-channel output I could use a single Zyn instance on a MIDI track, and then spread the outputs into multiple buses to process each part separately.

What do you think?

Also: could there be a way to switch betwen the two without having to manually save/load XMZ files?

**Comment by fundamental** [comment]
> I wonder if there could be a plug-in version that uses that?

Yes, though to do it properly would be somewhat non-trivial as some fairly brittle assumptions are made about buffer states to generate multiple outputs in the standalone.

> What do you think?

It's something I would imagine would appeal to a number of users, though it would require some effort to have in a reasonably polished state.

> Also: could there be a way to switch between the two without having to manually save/load XMZ files?

Unless I'm mistaken the number and type of inputs/outputs is fixed in the LV2 spec given a static .ttl file. There may be some extension to accomplish the goal, though without existing host support it seems like it would be a fringe case.

**Comment by Spunge** [comment]
I would love a multichannel version of the LV2 plugin.
